app/controllers/admin/orders_controller.rb

```diff
@@ -0,0 +1,16 @@
+class Admin::OrdersController < Admin::BaseController
+  before_action :require_admin
```

Comment:
same here with removing the require admin since the base controller has the method

Reply:
Same as above, resolved